dmci/api/app.py

```diff
@@ -87,10 +90,7 @@ def post_delete(metadata_id=None):
             else:
                 return self._formatMsgReturn(err), 400
 
-            if err:
-                return self._formatMsgReturn(err), 500
-            else:
-                return self._formatMsgReturn(OK_RETURN), 200
+            return self._formatMsgReturn(OK_RETURN), 200
```

Comment:
Assumes distributor ALWAYS succeeds.
self._distributor_wrapper returns an err to check

Reply:
aha - I see it now..